pages/_app.js

```diff
@@ -11,6 +16,8 @@ const inter = Inter({
 export default function App({ Component, pageProps }) {
 	return (
 		<main className={`${inter.variable} font-sans`}>
+			<Navbar/>
```

Comment:
These should never be in the _app.js file. Update the develop branch and pull latest changes, then make your modifications, please. The place for components is index.js

Reply:
Azzam will take care of this.